mollie/api/objects/onboarding.py

```diff
@@ -0,0 +1,42 @@
+from .base import Base
+
+
+class Onboarding(Base):
+    @classmethod
+    def get_resource_class(cls, client):
+        from ..resources.onboarding import Onboarding as OnboardingResource
+        return OnboardingResource(client)
+
+    @property
+    def resource(self):
+        return self._get_property('resource')
+
+    @property
+    def name(self):
+        return self._get_property('name')
+
+    @property
+    def signed_up_at(self):
+        return self._get_property('signedUpAt')
+
+    @property
+    def status(self):
```

Comment:
When a field like `status` as some predefined values, we add convenience methods and constants for them: see https://github.com/mollie/mollie-api-python/blob/982a1a63e8217c6c1fe03c52fde6e09e3e544f9d/mollie/api/objects/subscription.py#L21